contents/blog/the-posthog-array-1-18-0.md

```diff
@@ -0,0 +1,188 @@
+---
+date: 2020-11-30
+title: Array 1.18.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+Our primary goals for this release were to iron out bugs and improve the user experience of our Beta features.
+
+As a result, we fixed **a whole lot of stuff**. We merged dozens of PRs with session recording fixes and improvements, and a dozen more with updates to our plugins functionality. We also improved things like event ingestion, the UX for feature flags, and our settings for both organizations and projects. You can read through the entire list of fixes [later in this post](#bug-fixes-and-performance-improvements), but beware: it's quite long.
+
+Finally, thank you to everyone who helped us out with feedback and contributions during this release cycle, you help us make PostHog better every day.
+
+## Release Notes
+
+> If you're self-hosting and want to upgrade for a better experience with nicer features, remember to [update your PostHog instance](/docs/configuring-posthog/upgrading-posthog).
+
+### [Improvements to posthog-js](https://github.com/PostHog/posthog-js)
+
+A new version of `posthog-js` is available and we recommend you to update if you've installed it via `npm`. Snippet users have access to the latest version by default.
+
+The new version includes a lot of bugfixes that improve our session recording feature, as well as is significantly lighter, having had [a lot of legacy code removed](https://github.com/PostHog/posthog-js/pull/128). 
+
+R.I.P. to the hundreds of lines of JavaScript that were removed - you will not be missed.
+
+### [Plugins are now available on Kubernetes deployments](https://github.com/PostHog/charts/pull/24)
+
+Following feedback from a user, we have now added support for [PostHog Plugins](/docs/plugins/overview) to our Helm chart. 
+
+If you're using the chart to deploy PostHog, upgrading to the latest version will give you access to the new plugin server (Beta).
+
+### [Honorary Mention: Requiring Curly Brackets](https://github.com/PostHog/posthog/pull/2505)
+
+Our codebase now disallows bracket-less control structures, enforcing curly brackets on if statements. 
+
+As described by Michael, one of our engineers:
+
+_"They don't do anything, yet we thank them for their existence."_
```

Comment:
Madman, you actually did it.

Reply:
Our release notes have always been lighthearted, plus you gave me a great quote for it